cmd/karavictl/cmd/storage_create.go

```diff
@@ -81,6 +89,12 @@ var storageCreateCmd = &cobra.Command{
 			}
 			return v
 		}
+		flagStringSliceValue := func(v []string, err error) []string {
+			if err != nil {
+				errAndExit(err)
+			}
```

Comment:
need to check for an empty slice here

Reply:
For registering all PowerMax arrays in Unisphere, this slice will be empty. For PowerFlex, we check the length here: 

https://github.com/dell/karavi-authorization/blob/79b8fc31e2d7c66b85447eebbe64d67ca2e0fb5d/cmd/karavictl/cmd/storage_create.go#L234